### PR TITLE
Add module map

### PR DESCRIPTION
**Pull request by JensAyton**

### CODE REVIEW

**Comment by JensAyton** [comment]
@spotify/objc-dev

**Comment by 5at5ish** [comment]
👍

File: include/SPTDataLoader/module.modulemap [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module SPTDataLoader {
+    umbrella "."
```
Comment (rastersize) [inline include/SPTDataLoader/module.modulemap]: Could we use the [`SPTDataLoaderConvenience.h`](https://github.com/spotify/SPTDataLoader/blob/master/include/SPTDataLoader/SPTDataLoaderConvenience.h) umbrella header instead?

Comment (dflems) [inline include/SPTDataLoader/module.modulemap]: It's a breaking change, but we should probably rename `SPTDataLoaderConvenience.h` to `SPTDataLoader.h` and rename the existing `SPTDataLoader.h` to... something else.

**Pull request merged**